backend/src/test/groovy/com/capstone/moneytree/config/DefaultStompFrameHandlerConfigTest.groovy

```diff
@@ -0,0 +1,24 @@
+package com.capstone.moneytree.config
+
+import org.junit.Test
+import spock.lang.Specification
+
+import java.util.concurrent.BlockingQueue
+import java.util.concurrent.LinkedBlockingDeque
+
+class DefaultStompFrameHandlerConfigTest extends Specification {
+
+    private DefaultStompFrameHandlerConfig defaultStompFrameHandler
+
+    @Test
+    def "It should initialize the STOMP frame handler"() {
+        given: "The frame handler"
+        BlockingQueue<String> blockingQueue = new LinkedBlockingDeque<>()
+
+        when: "we initialize the frame handler"
+        defaultStompFrameHandler = new DefaultStompFrameHandlerConfig(blockingQueue)
+
+        then: "should be initialized"
+        assert defaultStompFrameHandler != null
```

Comment:
It is up to you or not to remove it but within the `then` clause, you do not need to assert, the spock takes care of that for you :)

Reply:
Ah I did not know that! Thanks for the info :D